### PR TITLE
Typo, slight rephrase

### PR DESCRIPTION
**Pull request by KronosTheLate**

### CODE REVIEW

File: docs/src/examples/higher_order.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ eqs = [D(D(x)) ~ σ * (y - x),
...
+`D = Differential(t)^2` and then `D(x)` would be the second derivative,
 and this syntax extends to `N`-th order. Also, we can use `*` or `∘` to compose
 `Differential`s, like `Differential(t) * Differential(x)`.
 
 Now let's transform this into the `ODESystem` of first order components.
 We do this by simply calling `ode_order_lowering`:
 
 ```@example orderlowering
-sys = structural_simplify(sys)
+sys = ode_order_lowering(sys)
```
Comment (ChrisRackauckas) [inline docs/src/examples/higher_order.md:37]: no you should just use `structural_simplify` for this these days

File: docs/src/examples/higher_order.md [hunk 2]
```diff
@@ -26,15 +26,15 @@ eqs = [D(D(x)) ~ σ * (y - x),
 ```
 
 Note that we could've used an alternative syntax for 2nd order, i.e.
-`D = Differential(t)^2` and then `E(x)` would be the second derivative,
+`D = Differential(t)^2` and then `D(x)` would be the second derivative,
 and this syntax extends to `N`-th order. Also, we can use `*` or `∘` to compose
 `Differential`s, like `Differential(t) * Differential(x)`.
 
 Now let's transform this into the `ODESystem` of first order components.
 We do this by simply calling `ode_order_lowering`:
```
Comment (YingboMa) [inline docs/src/examples/higher_order.md:34]: We should change this instead :p

File: docs/src/examples/higher_order.md (hunk 1 above)
Comment (KronosTheLate) [reply docs/src/examples/higher_order.md]: done

File: docs/src/examples/higher_order.md (hunk 2 above)
Comment (KronosTheLate) [reply docs/src/examples/higher_order.md]: I don't really get what you mean... 😅

**Comment by KronosTheLate** [comment]
As a sidenote, if `sys = structural_simplify(sys)` is as common as it seems, perhaps it would make sense to make a mutating version `structural_simplify!`.

**Pull request merged**